pkg/vsphere/session/session.go

```diff
@@ -328,3 +345,49 @@ func isNotAuthenticated(err error) bool {
 	}
 	return false
 }
+
+// GetESXSession returns a general-purpose ESX session for tests.
+func GetESXSession(ctx context.Context, service string) (*Session, error) {
```

Comment:
Just had a conversation with @anchal-agrawal for this code change. Cause that is to make the test code easier to use vcsim, it might be better to see some helper function in vcsim, but now vcsim is moved out of vic repo.
So far, the suggestion is to move it into https://github.com/vmware/vic/blob/master/pkg/vsphere/test/test.go, where the package is only used by test code.
How do you think?